### DISCUSSION TITLE
ctrl + / used to comment a line

### DISCUSSION THREAD

**Discussion by danmartinvela** [zed-industries/zed#12843] [Feature Requests]
A month ago ctrl + / used to comment lines... now it's completely gone... is there any way to retrieve that behavior?

**Comment by Moshyfawn** [comment]
What language do you see this happening in?

**Reply by danmartinvela** [reply]
C, I have been working with C lately

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
